### PR TITLE
iOS Keychain conformance

### PR DESCRIPTION
**Pull request by heyzooi**
#### Description

JS SDK using the same params used by the Swift SDK

#### Changes

Using the same query params used by our Swift SDK

### CODE REVIEW

**Comment by heyzooi** [comment]
this PR is not required to fix the keychain issue, but conforms to our Swift SDK by using the same params.

**Comment by georgiwe** [comment]
I don't fully understand keychain and iOS in general, but I'm thinking "will there be a backwards compatibility issue"? Users would have to log in again? Is there any necessary clean up of the old stuff in the keychain or something? Can't we "migrate" those?

**Comment by thomasconner** [comment]
@heyzooi @georgiwe Georgi is right. These changes will require a user to login again. I think these changes make sense so that we use the same params as our Swift SDK but we need to think of how we will migrate existing active users. I will the missing backwards compatibility code.

**Comment by heyzooi** [comment]
right, let me refactor that, that's really no need to keep the same keys, but the security changes are more important to be in place

File: src/core/client.js [hunk 1]
```diff
@@ -158,7 +158,11 @@ export class Client {
    * Set the active user
    */
   setActiveUser(activeUser) {
-    return this.activeUserStorage.set(`${this.appKey}.${ACTIVE_USER_KEY}`, activeUser);
+    if (activeUser != null) {
```
Comment (heyzooi) [inline src/core/client.js]: question that I had was if should use `!=` or `!==` in this case

Comment (thomasconner) [reply src/core/client.js]: you can just do `if (activeUser) {`

Comment (heyzooi) [reply src/core/client.js]: done! thanks @thomasconner

**Comment by georgiwe** [comment]
I see this is approved - shouldn't we fix the unit tests first?

**Comment by tejasranade** [comment]
@thomasconner @heyzooi Who has the next steps on this PR? The unit tests are currently failing. We can't merge until they are fixed. Maybe this needs a JIRA?

**Comment by thomasconner** [comment]
We need to look into why the unit tests are failing and make any changes that are needed. I don't know what changes are needed if any at the moment.

**Comment by heyzooi** [comment]
@thomasconner should I merge this?

**Comment by heyzooi** [comment]
@thomasconner should I rebase to see if the unit tests are not working?

**Comment by heyzooi** [comment]
unit test are running again, @thomasconner i think you can now approve

**Pull request merged**